src/pages/post/Post.tsx

```diff
@@ -0,0 +1,193 @@
+                About
+              </NavigationMenuLink>
+            </NavigationMenuItem>
+            <NavigationMenuItem className="mx-4 w-40">
+              <NavigationMenuLink className={navigationMenuTriggerStyle()}>
+                Profile
+              </NavigationMenuLink>
+            </NavigationMenuItem>
+          </NavigationMenuList>
+        </NavigationMenu>
+      </div>
+      <div className="col-span-1 flex justify-evenly align-middle">
+        <Link to="">
+          <Avatar>
+            <AvatarImage src="random_link" />
+            <AvatarFallback>
+              <Icon icon="material-symbols-light:face-6" className="text-2xl" />
+            </AvatarFallback>
+          </Avatar>
+        </Link>
+        <Button className="flex justify-evenly align-middle">
+          <Icon icon="material-symbols-light:login" className="text-2xl" />
+        </Button>
+      </div>
+    </div>
+  );
+};
+
+export const Post: FC = () => {
+  // TODO: implement providers
+  // TODO: implement error page if user or post is not found
+  const currentPost = posts.filter((post) => post.id === "post_1")[0];
+
+  const postAuthor = users.filter(
+    (user) => user.id === currentPost?.user_id
+  )[0];
+
+  return (
+    <div className=" overflow-hidden bg-white">
+      <NavBar />
+      <div className="mb-4 mt-40 flex w-screen justify-center p-11 md:mt-20">
+        <p className="font-optical-sizing-auto normal font-title text-6xl font-bold">
+          {currentPost?.title}
+        </p>
+      </div>
+      <div className="flex w-screen justify-center">
+        <div className="grid w-[500px] grid-cols-3">
+          <div className="flex items-center justify-center">
+            <Icon icon="material-symbols-light:face-6" className="text-2xl" />
+            <span className="mx-1">
+              by <strong>{postAuthor.first_name}</strong>
+            </span>
+          </div>
+          <div className="flex items-center justify-center">
+            <Icon
+              icon="material-symbols-light:nest-clock-farsight-analog-outline"
+              className="text-2xl"
+            />
+            <span className="mx-1">{`${calculateReadTime(currentPost?.content || "").minutes} min read`}</span>
+          </div>
+          <div className="flex items-center justify-center">
+            <Icon
+              icon="material-symbols-light:calendar-month-outline"
+              className="text-2xl"
+            />
+            <span className="mx-1">
+              {currentPost?.creation_datetime?.toLocaleDateString() ||
+                "No date provided"}
+            </span>
+          </div>
+        </div>
+      </div>
+      <div className="my-5 grid grid-cols-8">
+        <div className="col-span-1 h-[18.75rem]"></div>
+        <div className="col-span-6 grid grid-cols-8 gap-5">
+          <div className="col-span-8 h-[30rem]">
+            <img src={banner_image} className="h-full w-full object-cover" />
```

Comment:
not large enough

example of banner img: https://www.freecodecamp.org/news/the-biggest-changes-in-javascript-this-year/?ref=dailydev